src/atmos_param/rrtm_radiation/rrtm_radiation.f90

```diff
@@ -656,6 +656,7 @@ subroutine run_rrtmg(is,js,Time,lat,lon,p_full,p_half,albedo,q,t,t_surf_rad,tdt,
 	     ! Seasonal Cycle: Use astronomical parameters to calculate insolation
 	     call diurnal_solar(lat, lon, gmt, time_since_ae, coszen, fracsun, rrsun)
           end if
+          insolation = solr_cnst * rrsun
```

Comment:
has this been tested under the usual circumstances of no eccentricity?

Reply:
Yes